client/src/lib/parse/timeline/TimelineParser.ts

```diff
@@ -107,12 +110,71 @@ export default class TimelineParser {
     return allYears
   }
 
+  /**
+   * Returns the start and end index of the yearsRange to use for the timeline graph's brush.
+   * Calculated based on Interquartile Range (IQR)
+   *
+   * @param { Array<string> } sortedKeys
+   * @param { ITimelinesTransformed | null } transformedData
+   * @param { Array<string> } yearsRange
+   * @returns { initialStart: number; initialEnd: number }
+   */
+  static getStartAndEndIndex(
```

Comment:
@clarkepeterf Can you write a unit test for this function? I should already have mocked data for this.

Reply:
Done!